buildpacks/Makefile

```diff
@@ -28,5 +28,5 @@ all .PHONY: buildpacks
 buildpacks.clean:
 clean .PHONY: buildpacks.clean
 
-buildpacks.test:
+buildpacks.tests:
```

Comment:
I think this was a typo

Can someone clarify what this syntax does when it's by itself with nothing after the colon?

Reply:
@andrew-su

Reply:
Oops, thought I responded to this, sorry!

This was added here to let you know that this target should be inside every buildpack (additionally helps with autocomplete). Think of it like a placeholder.